app/src/main/java/com/x8bit/bitwarden/ui/platform/components/image/BitwardenGifImage.kt

```diff
@@ -0,0 +1,51 @@
+package com.x8bit.bitwarden.ui.platform.components.image
+
+import android.content.res.Configuration
+import androidx.annotation.DrawableRes
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalConfiguration
+import com.bumptech.glide.integration.compose.ExperimentalGlideComposeApi
+import com.bumptech.glide.integration.compose.GlideImage
+
+/**
+ * A composable that displays a gif image.
+ *
+ * The content will also be scaled to fit the image to the container.
+ *
+ * **NOTE:** Due to using Glide under the hood, this does not respect the themed resource management
+ * of the passed in [resId] automatically. If there is a specific version of the `.gif` you can pass
+ * in the optional [darkModeResId]. You would need to save these in your resources with unique
+ * names.
+ *
+ * @sample
```

Comment:
Without the changes mentioned here: 

https://github.com/user-attachments/assets/8eec0d81-a1ad-4084-96ef-292717fcfd1e